driver-scala/build.gradle

```diff
@@ -117,7 +117,7 @@ ext {
 }
 
 afterEvaluate {
-    jar.manifest.attributes['Automatic-Module-Name'] = 'org.mongodb.scala.mongo-scala-driver'
+    jar.manifest.attributes['Automatic-Module-Name'] = 'org.mongodb.scala.driver'
```

Comment:
This one seems inconsistent with the others, where `driver` is before the language.  Should we change to `org.mongodb.driver.scala`?

Reply:
I aimed for consistency with the archive name, following Java's approach when the Automatic Module name is missing. However, for overall consistency, it is better to stick with having `driver` before the language. I will rename it. Thanks!